pdp/storage.go

```diff
@@ -143,15 +143,17 @@ func (c *command) describe() string {
 		}
 	}
 
+	opPath := strings.Join(qpath, "/")
 	if nil != c.entity {
 		if evaluable, ok := c.entity.(Evaluable); ok {
-			if id, ok := evaluable.GetID(); ok {
-				qpath = append(qpath, strconv.Quote(id))
-			}
+			// change to something on path
```

Comment:
This and one below are both misleading comments. Please remove or rewrite them. We don't change path or something on path. We always change policies we have. The difference is that some updates contain entity we want to bring (as for "add") and for other we use entity which we already have (as for "del").
Additionally, what is more important, it makes your messages asymmetrical. It was symmetrical before because both didn't expose item in question only its location. Now it should as well say something like that:
"Adding/Deleting policy_set/policy/rule with following policies/rules at ...". I understand that getting information for delete is challenging for delete operation but still think that it's more natural to make them symmetrical.